src/runtime_src/core/tools/xbutil2/xrt-smi

```diff
@@ -35,7 +35,7 @@ XRT_LOADER_DIR="`dirname \"$0\"`"
 
 # For edge platforms loader is not required as tools are in standard location(/usr).
 # So calling unwrapped tool from this script itself.
-if [[ $XRT_LOADER_DIR =~ "/usr" || $XRT_LOADER_DIR =~ "/bin" ]]; then
+if [[ $XRT_LOADER_DIR =~ "/usr" ]]; then
```

Comment:
Hmm, doesn't this break what you fixed in #8426 ?

Reply:
Ah, yes, I'll need to discuss with Rahul on how to fix this for them.